### PR TITLE
[ci] bring over useful scorecard features to ci user page

### PR DESCRIPTION
**Pull request by daniel-goldstein**
CI user page now contains assigned PRs and random team member selection. These seem to me like the only features of scorecard people use and potentially eliminates the need for scorecard altogether. The CSS is a little crude but I'm hoping it's temporary, also happy to take suggestions.

### CODE REVIEW

File: ci/ci/constants.py [hunk 1]
```diff
@@ -1,32 +1,34 @@
...
+from typing import Optional, List
 
 GITHUB_CLONE_URL = 'https://github.com/'
 GITHUB_STATUS_CONTEXT = 'ci-test'
+TEAMS = ['Services', 'Compiler']
 
 
 class User:
-    def __init__(self, gh_username: str, hail_username: Optional[str]):
+    def __init__(self, gh_username: str, hail_username: Optional[str] = None, teams: List[str] = []):
```
Comment (danking) [inline ci/ci/constants.py:9]: In situations like this, we tend to stick a `*` before `hail_username` to force the subsequent arguments to be keyword-only. I suppose in this case you'll never have a teams and not a hail_username, so this is probably the right choice actually.

File: ci/ci/ci.py [hunk 2]
```diff
@@ -207,28 +209,38 @@ async def get_job(request, userdata):
...
+    team_random_member = {
+        team: random.choice([user for user in AUTHORIZED_USERS if team in user.teams]).gh_username for team in TEAMS
+    }
+
     page_context = {
         'username': username,
         'gh_username': gh_username,
-        'watched_branches': wbs,
+        'pr_wbs': pr_wbs,
+        'review_wbs': review_wbs,
```
Comment (danking) [inline ci/ci/ci.py:242]: So this is great, but scorecard does something different. Scorecard tells you things that you need to take action on. That means:
- your PRs that are failing tests
- your PRs that have changes requested reviews
- PRs assigned to you that you have no extant review (e.g. it got dismissed, you haven't reviewed yet)

**Review by danking — APPROVED** [review]
Love it, big improvement. See my comment about, I guess, an "action items" list.

File: ci/ci/ci.py (hunk 2 above)
Comment (daniel-goldstein) [reply ci/ci/ci.py:242]: Ya this does take a different approach, but I think the information is still there. For any PR, it shows the build state and review state, so you can see which PRs are failing tests and which PRs have review pending / changes requested / approved for your PRs as well as those assigned to you.

Comment (daniel-goldstein) [reply ci/ci/ci.py:242]: Or do you mean that it would be nice to eliminate PRs which don't require action?

**Pull request merged**